zkevm-circuits/src/evm_circuit.rs

```diff
@@ -94,6 +96,18 @@ impl<F: Field> SubCircuitConfig<F> for EvmCircuitConfig<F> {
             &exp_table,
         ));
 
+        meta.annotate_lookup_any_column(byte_table[0], || "Byte lookup column");
+        fixed_table.iter().enumerate().for_each(|(idx, &col)| {
+            meta.annotate_lookup_any_column(col, || format!("Fix table col {}", idx))
+        });
```

Comment:
I have a small suggestion: to always use a single word for a column annotation (no spaces).
So these could be `byte_range` and `fix_table_{}`.
I think this will help with more compact errors, and also opens the possibility to use the annotations in expressions like: `column_a * byte_range - 1`

Reply:
Fixed in 5b537c2c